### ISSUE TITLE
Doesn't work in IE11

### ISSUE THREAD

**Issue by glombek** [termi/Element.details#1]
Does work though when emulating IE10 or below

**Comment by glombek** [comment]
Element.details.js line 61

IE 11 seems to always return e.details as 0.

**Comment by Nettsentrisk** [comment]
Yep, discovered the same here. Any of these details/summary polyfills that work properly?

**Comment by MattiSG** [comment]
+1
